src/pages/Quiz.tsx

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { RootState } from '../App';
-import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import { fetchQuestions } from '../reducers/quiz';
 
 const Quiz = () => {
+	const dispatch = useDispatch();
+	const navigate = useNavigate();
 	const name = useSelector((state: RootState) => state.user.name);
+	const questions = useSelector((state: RootState) => state.quiz.questions);
+
+	useEffect(() => {
+		dispatch(fetchQuestions());
+	}, [dispatch]);
+
+	const startGame = () => {
+		if (questions) {
+			navigate('/quiz/0');
+		}
+	};
```

Comment:
Use a `Link` component instead.

Show a text saying "Loading questions" if the fetch is pending.